frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -23,20 +21,34 @@ Feature: Applications managed by GitOps and Create secret
              Then user will see the list GitOps application groupings on the page
 
         @regression @manual
-        Scenario: Application Details page for Applications: GO-01-TC04
+        Scenario: Application Details page for Applications: GO-01-TC03
             Given user is on the Environments page
               And user can see the Applications on the page
              Then user can add new environments using kam cli
 
         @regression @manual
-        Scenario: Application Details page for Applications: GO-01-TC03
+        Scenario: Application Details page for Applications: GO-01-TC04
+            Given user is on the Environments page
+              And user can see the Applications on the page
+             Then user can see various environments for that application
+              And user can see "overview" and "Deployment history" tab for that application
+              And user can see "dev", "stage" and "prod" workloads that are deployed in the environment
```

Comment:
won't necessarily be "dev", "stage" and "prod", can be anything they name it

Reply:
I converted it into a more generic sentence. Any number of workload will be taken care by gherkin + cypress implementation